Corona-Warn-App/src/main/res/layout/trace_location_organiser_qr_codes_list_item.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/main_box"
+    style="@style/ContactDiaryExpandableListItem"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <de.rki.coronawarnapp.ui.eventregistration.common.TraceLocationCardHighlightView
+        android:id="@+id/icon"
+        android:layout_width="88dp"
+        android:layout_height="136dp"
+        android:layout_margin="16dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="21.01.2021">
+
+        <ImageView
+            android:layout_width="42dp"
+            android:layout_height="42dp"
+            android:importantForAccessibility="no"
+            android:src="@drawable/ic_qr_code_list_item_icon"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+    </de.rki.coronawarnapp.ui.eventregistration.common.TraceLocationCardHighlightView>
```

Comment:
### Usage

Set text as caption by using `android:text` on `TraceLocationCardHighlightView`, and any qr codes or other text views that should be displayed on top of the gradient, can be added as child of `TraceLocationCardHighlightView`.

The container with the gradient as background has a fixed size, while the caption text has a fixed width, but will grow in height to wrap it's content (i.e. line breaks possible).